frontend-react/e2e/spec/all/public/about/case-studies.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { AboutCaseStudiesPage } from "../../../../pages/public/about/case-studies";
+import { test as baseTest, expect } from "../../../../test";
+
+export interface Fixtures {
+    aboutCaseStudiesPage: AboutCaseStudiesPage;
+}
+
+const test = baseTest.extend<Fixtures>({
+    aboutCaseStudiesPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new AboutCaseStudiesPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe(
+    "About / Case Studies page",
+    {
+        tag: "@smoke",
+    },
+    () => {
+        test("has correct title", async ({ aboutCaseStudiesPage }) => {
```

Comment:
You do not need this section since this code below already tests for that:

```
test("has correct title + heading", async ({ aboutCaseStudiesPage }) => {
  await aboutCaseStudiesPage.testHeader();
});
```

Reply:
This applies to `frontend-react/e2e/spec/all/public/about/release-notes.spec.ts` as well.